scripts/src/tests/helpers.ts

```diff
@@ -170,6 +174,22 @@ export async function clearDatabase() {
 	}
 }
 
+const CONFIG = getConfig();
+const PRIVATE_KEYS = readKeysDir(path.join(CONFIG.jwt_keys_dir, "private_keys"));
+const PUBLIC_KEYS = readKeysDir(path.join(CONFIG.jwt_keys_dir, "public_keys"));
```

Comment:
I think @nitzantomer would like us to move these consts to the top of the file

Reply:
yeah, it makes more sense to have all variable declarations to the top of the module